src/layout/section/tripsSection/index.tsx

```diff
@@ -24,6 +24,8 @@ const StyledTripsSection = styled(TripsSection)`
 
   & .kirk-tripsSection .kirk-tripCard {
     scroll-snap-align: start;
+    max-width: 50%;
```

Comment:
Need to remove or restrict this rule. We can have only one TopTripCard (cheapest & closest) that takes 100%